libraries/botbuilder-ai/tests/qnaMaker.test.js

```diff
@@ -778,6 +780,105 @@ describe('QnAMaker', function () {
             assert(typeof fetch === 'function');
         });
     });
+
+    describe('Active Learning', function() {
+        it('executeHttpRequest() should return JSON result from Train API response of 204 No-Content', function() {
+            const { endpoint, trainApi, feedbackRecords, successfullyTrainedResult } = new TrainApiHelper();
+
+            async () => await executeHttpRequestToTrainApi();
+        
+            async function executeHttpRequestToTrainApi() {
```

Comment:
Please move the arrangement to before the act in the test.

Reply:
Sorry for the wonky test arrangement. I was running into the "await can only be used in an async function" error earlier, *completely* forgetting that the mocha test cases are encapsulated within a function already which I could easily make async 🤦 🤦 Was doing backflips trying to wrap things in random async helper functions 😅 

Made callback in `it()` `async`, so no helper function needed => cleaner, and tests are actually arranged properly now 👍